.pre-commit-config.yaml

```diff
@@ -1,8 +1,26 @@
 repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v5.0.0
+    hooks:
+      - id: check-case-conflict
+      - id: check-yaml
+      - id: check-shebang-scripts-are-executable
+      - id: check-json
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
   - repo: local
     hooks:
       - id: org-sort
         name: org-sort
         entry: ./org-sort.sh
         language: system
         pass_filenames: false
+  - repo: https://gitlab.cee.redhat.com/infosec-public/developer-workbench/tools.git
```

Comment:
Any concerns exposing internal resources?

Reply:
I have no concerns about it - it's behind the VPN and open to any RH's. If your a non-RH'er, you'd need to disable the hook but who would that be? as we don't see that currently.

whats your worry/concern, if any?

Reply:
Not a big concern. The only point that _may_ require calling out is that there is another step (not documented) that needs to be completed prior to being able to leverage this tool (authenticating with `rh_gitleaks`) first

Reply:
added blurb to `README.md`